TEST_APPS/testcases/netsocket/TCPSOCKET_ACCEPT.py

```diff
@@ -0,0 +1,102 @@
+"""
+Copyright 2018 ARM Limited
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import threading
+import time
+
+from mbed_clitest.bench import Bench
+from interface import interfaceUp, interfaceDown
+from mbed_clitest.tools import test_case
+from mbed_clitest.TestStepError import SkippedTestcaseException
+
+class Testcase(Bench):
+    def __init__(self):
+        Bench.__init__(self,
+                       name="TCPSOCKET_ACCEPT",
+                       title = "TCPSOCKET_ACCEPT",
+                       purpose = "Test that TCPSocket::bind(), TCPSocket::listen() and TCPSocket::accept() works",
+                       status = "released",
+                       component= ["mbed-os", "netsocket"],
+                       author = "Juha Ylinen <juha.ylinen@arm.com>",
+                       type="smoke",
+                       subtype="socket",
+                       requirements={
+                           "duts": {
+                               '*': { #requirements for all nodes
+                                    "count":2,
+                                    "type": "hardware"
+                                },
+                                "1": {"nick": "dut1"},
+                                "2": {"nick": "dut2"}
+                           }
+                       }
+        )
+
+    def setup(self):
+        interface = interfaceUp(self, ["dut1"])
+        self.server_ip = interface["dut1"]["ip"]
+        interface = interfaceUp(self, ["dut2"])
+        self.client_ip = interface["dut2"]["ip"]
```

Comment:
This is copy&paste from old test case.. use `ifup`

Reply:
I am not sure how to extract the IP address of the interface from the ifup command. And we need to know the ip address to call the connect command a few lines later.
As far as I can see the IP address will only be printed on the console...